src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -117,14 +129,27 @@ def compute_reference(frames: np.ndarray,
             frame[:] = shift_frame(frame=frame, dy=dy, dx=dx)
 
         # Create a new reference image from the highest correlated data.
-        ref_image = max_corr_frames.mean(axis=0).astype(np.int16)
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', 'Mean of empty slice')
+            ref_image = np.nanmean(max_corr_frames, axis=0)
         # Shift reference image to position of mean shifts to remove any bulk
         # displacement.
         ref_image = shift_frame(
```

Comment:
Sorry if I'm missing something. Couldn't this still wrap given that it uses `np.roll` under the hood?

Then on line 144, we would be cutting off some of the frame?